recipe-server/client/control/state/actions/actions.js

```diff
@@ -0,0 +1,43 @@
+import {
+  ACTION_RECEIVE,
+} from '../action-types';
+
+import {
+  makeApiRequest,
+} from '../requests/actions';
+
+
+function receiveAction(action) {
+  return dispatch => {
+    dispatch({
+      type: ACTION_RECEIVE,
+      action,
+    });
+  };
+}
+
+
+export function fetchAction(name) {
+  const requestId = `fetch-${name}`;
+  return makeApiRequest(requestId, `action/${name}/`, { method: 'GET' })
+    .then(action => receiveAction(action));
```

Comment:
Is `receiveAction` used anywhere besides here? I think breaking this out into its own method reduces readability.